steamos-waydroid-installer.sh

```diff
@@ -483,3 +507,8 @@ EOF
 	echo -e "$current_password\n" | sudo -S steamos-readonly enable
 	echo Waydroid has been successfully installed!
 fi
+if zenity --question --text="Do you Want to Return to Gaming Mode?"; then
+              qdbus org.kde.Shutdown /Shutdown org.kde.Shutdown.logout
+            else
+              echo OK!
+            fi
```

Comment:
indentation is incorrect here as well; none needed for `else` and `fi` and should be tabs, not spaces

for stylistic consistency, should also have a new line before the question, splitting up the block. and newline at EoF

```suggestion

if zenity --question --text="Do you Want to Return to Gaming Mode?"; then
	qdbus org.kde.Shutdown /Shutdown org.kde.Shutdown.logout
else
	echo OK!
fi

```

Reply:
actually I don't think the "OK!" is needed either in this case; can just remove the `else` block entirely
```suggestion

if zenity --question --text="Do you Want to Return to Gaming Mode?"; then
	qdbus org.kde.Shutdown /Shutdown org.kde.Shutdown.logout
fi

```